src/legacy_stm32_flash_provider.erl

```diff
@@ -24,10 +24,10 @@
 -export([init/1, do/1, format_error/1]).
 
 -define(PROVIDER, stm32_flash).
--define(DEPS, [packbeam]).
+-define(DEPS, [{default, app_disovery}]).
```

Comment:
I don't now if this is correct. Does `app_disovery` make sure the application has been compiled, and packed into an avm file?

Reply:
Rats I thought I fixed that.  Will fix.